app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -35,24 +38,30 @@ class FeedbackSummaryContainer extends React.Component {
     }, []);
     actions.feedback.setFeedback(feedbackItems);
   }
+
+  render() {
+    return (this.props.feedback.length) ? <FeedbackSummary feedback={this.props.feedback} /> : null;
```

Comment:
You're right that `this.props.feedback.length > 0` would be more explicit, rather than relying on implicit conversion of a number to a boolean.